crates/store/src/db/schema.rs

```diff
@@ -103,8 +103,8 @@ diesel::table! {
 diesel::joinable!(accounts -> account_codes (code_commitment));
 diesel::joinable!(accounts -> block_headers (block_num));
 // Note: Cannot use diesel::joinable! with accounts table due to composite primary key
-// diesel::joinable!(notes -> accounts (sender));
-// diesel::joinable!(transactions -> accounts (account_id));
+// diesel::joinable!(notes -> accounts (sender)); diesel::joinable!(transactions -> accounts
+// (account_id));
```

Comment:
This is one outlier which is "malformatted" but I think likely we should place this in a code fence or something.

I'm leaving this as is for now just for discussion's sake.